data/tooltips.txt

```diff
@@ -81,6 +81,9 @@ tip "cargo scan power:"
 	`The range of this cargo scanner is proportional to the square root of this value.`
 
 tip "cargo scan speed:"
+	`A higher scan speed will correspond to a quicker scan.`
```

Comment:
I'd say we should revert these tooltips to their old versions before the scan speed changes, but the old tooltips aren't true anymore since we're converting to efficiency.
> Determines how long it takes to perform a cargo scan. A total value of 1 means one second, and the time decreases in proportion to the square root of this value, e.g. 4 means half a second.

I think at least taking the first sentence would be better, though.
```suggestion
	`Determines how long it takes to perform a cargo scan. A higher value will correspond to a quicker scan.`
```